xds/src/main/java/io/grpc/xds/XdsLoadReportClientImpl.java

```diff
@@ -263,11 +268,12 @@ private void handleResponse(LoadStatsResponse response) {
       List<String> serviceList = Collections.unmodifiableList(response.getClustersList());
       // For gRPC use case, LRS response will only contain one cluster, which is the same as in
       // the EDS response.
-      if (serviceList.size() != 1 || !serviceList.get(0).equals(serviceName)) {
-        logger.log(ChannelLogLevel.ERROR, "Unmatched cluster name(s): {0} with EDS response: {1}",
-            serviceList, serviceName);
+      if (serviceList.size() != 1) {
+        logger.log(ChannelLogLevel.ERROR, "Received clusters: {0}, expect exactly one",
+            serviceList);
         return;
       }
+      clusterName = serviceList.get(0);
```

Comment:
`clusterName` can be a field of this inner class, as it is used only in the inner class.

Reply:
Done.